torch/_utils_internal.py

```diff
@@ -52,5 +53,42 @@ def get_source_lines_and_file(obj):
     return sourcelines, file_lineno, filename
 
 
+def check_module_version_greater_or_equal(module, req_version_tuple, error_if_malformed=True):
+    '''
+    Check if a module's version satisfies requirements
+
+    Usually, a module's version string will be like 'x.y.z', which would be represented
+    as a tuple (x, y, z), but sometimes it could be an unexpected format. If the version
+    string does not match the given tuple's format up to the length of the tuple, then
+    error and exit or emit a warning.
+
+    Args:
+        module: the module to check the version of
+        req_version_tuple: tuple (usually of ints) representing the required version
+        error_if_malformed: whether we should exit if module version string is malformed
+
+    Returns:
+        requirement_is_met: bool
+    '''
+    try:
+        version_strs = module.__version__.split('.')
+        # Cast module version fields to match the types of the required version
+        module_version = tuple(
+            type(req_field)(version_strs[idx]) for idx, req_field in enumerate(req_version_tuple)
+        )
+        requirement_is_met = module_version >= req_version_tuple
+
+    except:
+        message = ("'%s' module version string is malformed '%s' and cannot be compared"
+            " with tuple %s" % (module.__name__, module.__version__, str(req_version_tuple)))
+        if error_if_malformed:
+            raise Exception(message)
```

Comment:
`RuntimeError` is probably better here. You normally want something a little more specific than a plain `Exception`, because that's too generic to catch well (it's the base class for all exceptions).

Reply:
Good point. I changed it to RuntimeError. This function has been moved to serialization.py, BTW.